var/spack/repos/builtin/packages/braker/package.py

```diff
@@ -33,10 +33,12 @@ class Braker(Package):
     url      = "http://bioinf.uni-greifswald.de/augustus/binaries/BRAKER1_v1.11.tar.gz"
     list_url = "http://bioinf.uni-greifswald.de/augustus/binaries/old"
 
+    version('2.1.0', '5f974abcceb9f96a11668fa20a6f6a56',
+            url='http://exon.gatech.edu/Braker/BRAKER2.tar.gz')
```

Comment:
It looks like version 2.1.0 is also available at http://bioinf.uni-greifswald.de/augustus/binaries/BRAKER_v2.1.0.tar.gz, is there any reason to prefer the other URL?